org.eu.getcatalyst.Catalyst.json

```diff
@@ -0,0 +1,59 @@
+{
+  "app-id": "org.eu.getcatalyst.Catalyst",
+  "base": "org.electronjs.Electron2.BaseApp",
+  "base-version": "23.08",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "23.08",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "run.sh",
+  "separate-locales": false,
+  "finish-args": [
+    "--share=ipc",
+    "--share=network",
+    "--socket=fallback-x11",
+    "--socket=wayland",
```

Comment:
"--socket=fallback-x11" + "--socket=wayland" won't work. Electron does not default to wayland without a flag. You didn't pass a flag.

Reply:
Yes X11 is fine by me

Reply:
That makes wayland default. If you want x11 only, keep `--socket=x11` only.

Reply:
I don't care and have no Wayland test machine. I want Wayland to work for people who need it

Reply:
This does not make it work, pass `--ozone-platform-hint` or `--ozone-platform`

Reply:
It is now broken for Wayland environments

```
flatpak run org.eu.getcatalyst.Catalyst       
                                                       
[3:1230/191237.762368:ERROR:bus.cc(399)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
[3:1230/191237.960972:ERROR:ozone_platform_x11.cc(241)] Missing X server or $DISPLAY
[3:1230/191237.960983:ERROR:env.cc(255)] The platform failed to initialize.  Exiting.
/app/bin/run.sh: line 2:     3 Segmentation fault      (core dumped) zypak-wrapper /app/bin/catalyst
```

Reply:
Fixed, Wayland seems to work now